coherent_rates/plot.py

```diff
@@ -134,3 +137,42 @@ def plot_boltzmann_isf(
 
     fig.show()
     input()
+
+
+def plot_alpha_deltak(
+    system: PeriodicSystem,
+    config: PeriodicSystemConfig,
+    *,
+    k_points: list[int] | None = None,
+) -> None:
+    bound_data = get_ak_data_1d(system, config, k_points=k_points)
+    xpoints = bound_data["basis"].k_points
+    m, b = np.polyfit(xpoints, bound_data["data"], 1)
+    xfit = np.array([0, xpoints[len(xpoints) - 1]])
+    yfit = m * xfit + b
+    bound_mass = Boltzmann * config.temperature / (m * m)
```

Comment:
move this fitting step into another function, which calls a second function to do Boltzmann * config.temperature / (m * m)